README.md

```diff
@@ -130,6 +130,7 @@ The reporter provides a fine-grained configuration options through its builder t
                     .withJvmMetrics()
                     .withGlobalDimensions("Region=us-west-2", "Instance=stage")
                     .withZeroValuesSubmission()
+                    .withReportwithReportRawCountValue()
```

Comment:
Typo. You want `.withReportRawCountValue()`

Reply:
thanks